validateFileNames.js

```diff
@@ -0,0 +1,94 @@
+const shelljs = require('shelljs');
+
+// Regex that defines our file and directory naming convention
+const camelCasedFileNameRegex = /^[.]?([a-z])+([0-9]|[a-zA-Z]|[.])*$/;
+
+// Files and directories that are exempt from the naming convention
+const ignoredFilesAndDirectories = [
+  'README.md',
+  'Dockerfile',
+  'LICENSE.md',
+  'customTypings',
+  'typings',
+];
```

Comment:
@forabi I looked into using `find-packag-json`, it's a good idea but it still doesn't provide full protection without additional engineering. Like what if you were to run the script inside `functions/`? There's a `package.json` there, so it would fail.

I think it's fine to go with `process.cwd()` for now.